packages/calling/src/CallingClient/line/line.test.ts

```diff
@@ -0,0 +1,185 @@
+import {Mutex} from 'async-mutex';
+import {
+  getMobiusDiscoveryResponse,
+  getMockDeviceInfo,
+  getMockRequestTemplate,
+  getTestUtilsWebex,
+} from '../../common/testUtil';
+import {URL} from '../registration/registerFixtures';
+import {filterMobiusUris} from '../../common';
+import {
+  MobiusServers,
+  MobiusStatus,
+  ServiceIndicator,
+  WebexRequestPayload,
+} from '../../common/types';
+import {LineStatus} from './types';
+import Line from '.';
+import * as utils from '../../common/Utils';
+import SDKConnector from '../../SDKConnector';
+import {URL_ENDPOINT} from '../constants';
+import {LOGGER} from '../../Logger/types';
+import * as regUtils from '../registration/register';
+
+describe('Line Tests', () => {
+  const mutex = new Mutex();
+  const webex = getTestUtilsWebex();
+  SDKConnector.setWebex(webex);
+
+  const defaultServiceData = {indicator: ServiceIndicator.CALLING, domain: ''};
+  const createRegistrationSpy = jest.spyOn(regUtils, 'createRegistration');
+
+  const mobiusUris = filterMobiusUris(getMobiusDiscoveryResponse(), URL);
+  const primaryMobiusUris = jest.fn(() => mobiusUris.primary);
+  const backupMobiusUris = jest.fn(() => mobiusUris.backup);
+  const userId = webex.internal.device.userId;
+  const clientDeviceUri = webex.internal.device.url;
+
+  const handleErrorSpy = jest.spyOn(utils, 'handleRegistrationErrors');
+
+  jest.clearAllMocks();
+
+  describe('Registration tests', () => {
+    let line;
+    const mockRegistrationBody = getMockDeviceInfo();
+
+    const discoveryBody: MobiusServers = getMobiusDiscoveryResponse();
+    const primaryUrl = `${discoveryBody.primary.uris[0]}/calling/web/`;
+
+    const registrationPayload = <WebexRequestPayload>(<unknown>{
+      statusCode: 200,
+      body: mockRegistrationBody,
+    });
+
+    const mockBody = {
+      userId: webex.internal.device.userId,
+      clientDeviceUri: webex.internal.device.url,
+      serviceData: {
+        domain: '',
+        indicator: 'calling',
+      },
+    };
+
+    beforeEach(() => {
+      line = new Line(
+        userId,
+        clientDeviceUri,
+        LineStatus.ACTIVE,
+        mutex,
+        primaryMobiusUris(),
+        backupMobiusUris(),
+        LOGGER.INFO
+      );
+    });
+
+    afterEach(() => {
+      jest.clearAllTimers();
+      jest.clearAllMocks();
+      jest.useRealTimers();
+    });
+
+    it('verify successful Registration cases and keepalive', async () => {
+      jest.useFakeTimers();
+      webex.request.mockReturnValue(registrationPayload);
+
+      expect(createRegistrationSpy).toBeCalledWith(
+        webex,
+        defaultServiceData,
+        expect.any(Mutex),
+        expect.anything(),
+        LOGGER.INFO
+      );
+      expect(line.getRegistrationStatus()).toEqual(MobiusStatus.DEFAULT);
+      await line.register();
+
+      expect(webex.request).toBeCalledTimes(1);
```

Comment:
Can we change this to use toBeCalledOnceWith here? Applies everywhere where we are using toBeCalledTimes(1)

Reply:
Done at all other places and all other files as well.